### PR TITLE
feat: container monitoring addon supports other Azure clouds

### PR DESCRIPTION
**Pull request by ganga1980**
**Reason for Change**:

Container monitoring addon supported AzureChinaCloud and AzureUSGovernmentCloud in addition to Azure Public cloud. This PR has all required changes to work in AzureChinaCloud and AzureUSGovernmentCloud environments.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: cmd/deploy.go [hunk 1]
```diff
@@ -358,7 +358,21 @@ func autofillApimodel(dc *deployCmd) error {
 
 	if k8sConfig != nil && k8sConfig.Addons != nil && k8sConfig.IsContainerMonitoringAddonEnabled() {
 		log.Infoln("container monitoring addon enabled")
-		err := dc.configureContainerMonitoringAddon(ctx, k8sConfig)
+		var workspaceDomain string
+		cloudspecConfig := dc.containerService.GetCloudSpecConfig()
+		log.Infoln("selected cloud name:", cloudspecConfig.CloudName)
```
Comment (jackfrancis) [inline cmd/deploy.go]: Let's get rid of this log statement, as we don't need to output this for all cluster deploy operations

Comment (ganga1980) [reply cmd/deploy.go]: > Let's get rid of this log statement, as we don't need to output this for all cluster deploy operations

Fixed. removed the log statement.

**Comment by ganga1980** [comment]
@jackfrancis,  Thanks for reviewing this PR. I have fixed your comments. If you dont have any other feedback, please help on merging this PR.

**Pull request merged**